src/og/input/MouseHandler.ts

```diff
@@ -2,24 +2,26 @@
 
 class MouseHandler {
 
-    constructor(htmlObject) {
+    protected _htmlObject: HTMLElement;
+
+    constructor(htmlObject: HTMLElement) {
         this._htmlObject = htmlObject;
     }
 
-    setEvent(event, sender, callback) {
+    public setEvent(event: string, sender: any, callback: Function) {
```

Comment:
event here is <T extends Event> (lib.es event)

Reply:
or maybe MouseEvent

Reply:
If we are not sure, we should just use `any`. We can refine this in a further PR. Just a suggestion.